app/derivative_services/image_derivative_service.rb

```diff
@@ -13,16 +13,16 @@ def use=(use)
       @use = Array(use) + [Valkyrie::Vocab::PCDMUse.ServiceFile]
     end
 
-    def new(file_set)
-      ::ImageDerivativeService.new(file_set: file_set, original_file: original_file(file_set), storage_solution: storage_solution, image_config: image_config, use: use)
+    def new(model)
+      ::ImageDerivativeService.new(model: model, original_file: original_file(model), storage_solution: storage_solution, image_config: image_config, use: use)
     end
 
-    def original_file(file_set)
-      members(file_set).find { |x| x.use.include?(Valkyrie::Vocab::PCDMUse.OriginalFile) }
+    def original_file(model)
+      members(model).find { |x| x.use.include?(Valkyrie::Vocab::PCDMUse.OriginalFile) }
     end
 
-    def members(file_set)
-      adapter.query_service.find_members(model: file_set)
+    def members(model)
+      adapter.query_service.find_members(model: model)
```

Comment:
I think `model` is an instance everywhere except for the `find_all_of_model` query.